src/renderer/shared/ui/Icon/data/chevron.tsx

```diff
@@ -3,13 +3,17 @@ import RightImg, { ReactComponent as RightSvg } from '@shared/assets/images/chev
 import DownImg, { ReactComponent as DownSvg } from '@shared/assets/images/chevron/down.svg';
 import LeftImg, { ReactComponent as LeftSvg } from '@shared/assets/images/chevron/left.svg';
 import DropdownImg, { ReactComponent as DropdownSvg } from '@shared/assets/images/chevron/dropdown.svg';
+import DropdownDownImg, { ReactComponent as DropdownDownSvg } from '@shared/assets/images/chevron/dropdown-down.svg';
+import DropdownRightImg, { ReactComponent as DropdownRightSvg } from '@shared/assets/images/chevron/dropdown-right.svg';
 
 const ChevronImages = {
   up: { svg: UpSvg, img: UpImg },
   right: { svg: RightSvg, img: RightImg },
   down: { svg: DownSvg, img: DownImg },
   left: { svg: LeftSvg, img: LeftImg },
   dropdown: { svg: DropdownSvg, img: DropdownImg },
+  dropdownDown: { svg: DropdownDownSvg, img: DropdownDownImg },
```

Comment:
Maybe `shelfDown`? Or this name came from Figma?

Reply:
name from figma